packages/components/src/button/stories/index.js

```diff
@@ -1,35 +1,49 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
 /**
  * Internal dependencies
  */
 import Button from '../';
 
-export default { title: 'Button', component: Button };
+export default {
+	title: 'Button',
+	component: Button,
+};
+
+export const _default = () => {
+	const label = text( 'Label', 'Hello Button' );
```

Comment:
Should we do the same knob on each Button story?

Reply:
Sure, we can 👍

Reply:
Addressed in 34a2aa2:

![Screen Shot 2019-10-24 at 14 06 25](https://user-images.githubusercontent.com/699132/67484063-9433d500-f667-11e9-9b96-2f92dd8feef9.png)